src/services/severa-api-service.ts

```diff
@@ -675,3 +678,10 @@ const getSeveraAccessToken = async (): Promise<string> => {
     throw new Error(`Failed to get Severa access token: ${error.message}`);
   }
 };
+
+// Utility function to get correct Severa client ID based on environment
+const getSeveraClientId = (): string => {
+  return process.env.NODE_ENV === "production"
```

Comment:
I'm not certain that this will work as intended, as we have the staging environment which might also be classed as NODE_ENV === "production.
So we have 2 options, we handle this based on what envs are available in the pipeline, so we ensure only demo is in the staging pipeline and not demo id is in the production. This could work with a conditional selector for whichever one is truthy.

Doing at you are trying to do would be better, if we can confirm it is production in production but not in the staging env, perhaps the best way to check this is to add a log so we can see what the NODE ENV is when it's deployed to staging (vie develop merge).

Open to alternative suggestions